calicoctl/calico_ctl/status.py

```diff
@@ -252,3 +273,42 @@ def pprint_bird_protocols(version):
         print str(x) + "\n"
     else:
         print results + "\n"
+
+
+def _gobgp(cmd):
+    if getattr(sys, 'frozen', False):
+        gobgp = sys._MEIPASS + "/gobgp"
+    else:
+        gobgp = "gobgp"
+    results = subprocess32.check_output([gobgp, "-j"] + cmd.split(" "))
+    return json.loads(results)
+
+
+def pprint_gobgp_protocols(version):
+    x = PrettyTable(["Peer address", "Peer type", "State", "Since", "Info"])
+
+    now = int(time.mktime(datetime.datetime.now().timetuple()))
+    try:
+        neighbors = _gobgp("neighbor -t ipv%d" % version)
+    except subprocess32.CalledProcessError:
```

Comment:
@ishidawataru this is getting raised when I build and test the latest `node_image` and run the status command. 
PS: routes are getting installed fine, containers can also ping each other fine, the only problem is there is some gRPC connection problem while running the status command:
```bash

vagrant@calico-02:~/calico-containers$ sudo ./dist/calicoctl status --backend=gobgp
calico-node container is running. Status: Up 48 seconds
Running felix version 1.4.2.dev1

IPv4 BGP status
IP: 172.17.8.102    AS Number: 64511 (inherited)
2016/09/19 22:41:37 grpc: addrConn.resetTransport failed to create client transport: connection error: desc = "transport: dial tcp 127.0.0.1:50051: getsockopt: connection refused"; Reconnecting to {127.0.0.1:50051 <nil>}
Couldn't connect to gobgp.
IPv6 BGP status
No IPv6 address configured.
```